spring-cloud-gcp-docs/src/main/asciidoc/trace.adoc

```diff
@@ -32,20 +27,43 @@ dependencies {
 }
 ----
 
-You must enable Stackdriver Trace API from the Google Cloud Console in order to capture traces. Navigate to the
-https://console.cloud.google.com/apis/api/cloudtrace.googleapis.com/overview[Stackdriver Trace API] for your project
-and make sure it’s enabled.
+You must enable Stackdriver Trace API from the Google Cloud Console in order to capture traces.
+Navigate to the https://console.cloud.google.com/apis/api/cloudtrace.googleapis.com/overview[Stackdriver Trace API] for your project and make sure it’s enabled.
 
 A https://github.com/spring-cloud/spring-cloud-gcp/tree/{git_version}/spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample[sample application] is available.
 
 [NOTE]
 ====
-If you are already using a Zipkin server capturing trace information from multiple platform/frameworks, you also use a
-https://cloud.google.com/trace/docs/zipkin[Stackdriver Zipkin proxy] to forward those traces to Stackdriver Trace
-without modifying existing applications.
+If you are already using a Zipkin server capturing trace information from multiple platform/frameworks, you also use a https://cloud.google.com/trace/docs/zipkin[Stackdriver Zipkin proxy] to forward those traces to Stackdriver Trace without modifying existing applications.
 ====
 
+=== Tracing
+
+Spring Cloud Sleuth uses the https://github.com/openzipkin/brave[Brave tracer] to generate traces.
+This integration enables Brave to use the https://github.com/openzipkin/zipkin-gcp/tree/master/propagation-stackdriver[`StackdriverTracePropagation`] propagation.
+
+A propagation is responsible for extracting trace context from an entity (e.g., a HTTP servlet request) and for injecting trace context into an entity.
```

Comment:
"an HTTP"

Reply:
I think "a HTTP" request is actually the correct english, since it's not a mute H.

Reply:
http://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html